guillotina/files/manager.py

```diff
@@ -231,26 +232,30 @@ def __init__(self, context, request, field):
                     )
                 await download_resp.write(chunk)
                 await download_resp.drain()
+        except Response:  # pragma: no cover
+            http_exception = True
+            raise
```

Comment:
Why we don't want to raise all Exceptions?

Reply:
We do, this is to prevent the finally block from starting a response with the incorrect content length!

Reply:
Hmm, we only do for exceptions of type `Response` not all `Exceptions`

Reply:
gotcha, I think you're right.